RPIPs/RPIP-46.md

```diff
@@ -1,108 +0,0 @@
-   1. If shares change between claims, distributions MUST make an effort to account for the different values. For example, a distribution could use a duration-weighted average share. Approximations MAY be used where they significantly reduce complexity and/or costs.
-   2. Legacy minipools are an exception and MAY continue to support earlier distribution methodologies 
-6. `node_operator_commission_share`, `surplus_share`, `increase_no_share_seal_increment`, `increase_no_share_seal_count`, and `allowlisted_controllers` MAY be updated by pDAO vote
-7. `node_operator_commission_share` and `surplus_share`, MAY be updated by an address in the `allowlisted_controllers` array
-   1. This functionality SHALL not be used without a separate pDAO vote to enable a controller and add it to the list
-8. The security council SHALL have a limited-use power to increase the `node_operator_commission_share` by `increase_no_share_seal_increment` and decrease the `surplus_share` by the same amount
-   1. This power SHALL be usable if `increase_no_share_seal_count` > 0
-   2. `increase_no_share_seal_count` SHALL be decremented by one upon using this power
-   3. This power SHOULD be used if the deposit pool is over half-full for the majority of a 2-week period at the current `node_operator_commission_share`
-   4. The pDAO MAY change `increase_no_share_seal_count` via vote
-9. The initial settings SHALL be:
-   1. `node_operator_commission_share`: 3.5%
-   2. `voter_share`: 5%
-   3. `surplus_share`: 5.5%
-   4. `increase_no_share_seal_increment`: 0.5%
-   5. `increase_no_share_seal_count`: 6
-   6. `allowlisted_controllers`: []
-10. Prior to the release of Saturn 1, a ranked-choice vote MUST be held to select a mechanism for surplus revenue share
-    1. The choices MAY include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md), or using higher `voter_share`
-    2. If higher `voter_share` is selected, the entirety of the ["Specification taking effect with Saturn 2"](#specification-taking-effect-with-saturn-2) section below SHALL be deleted
-    3. The selected mechanism MAY be implemented in Saturn 1
-       1. If not, the revenue SHALL be held in reserve until the mechanism is implemented. In such a case, once the mechanism is implemented, the revenue SHALL be distributed in the same amount of time it took to build up, or faster.
-    4. The selected mechanism MUST be implemented in Saturn 2
-    5. When implemented, `surplus_share` SHOULD be renamed to something more specific
-
-## Specification taking effect with Saturn 2
-### Updating `voter_share`:
-The `voter_share` SHALL be adjustable, to target a share of voting eligible RPL (including 
-megapool staked RPL and legacy staked RPL) relative to the total supply of RPL minus RPL 
-owned by the protocol in its treasury or as a result of `surplus_share` buy backs. The voting
-eligible RPL target is a range between `vote_eligible_target_min` and `vote_eligible_target_max`
```

Comment:
Defining a maximum target for actively voting RPL seems wrong from a governance POV. General metric of most governance systems is their level of turnout, with higher almost always being better. At the least, should it be higher than 65%? Like 90%? 95%?

Reply:
We won't have an easy metric for turnout, let alone "informed turnout". The 65% came from a tiny straw poll. I think this is actually a good candidate for the pDAO to select itself -- a sentiment poll could work, or even a full vote. This number doesn't dramatically impact the rest. There is a direct tradeoff with RPL having a shot at getting exogenous value at some point (eg, being usable as collateral for a loan).

Personally, 95% feels too high. 65-90% all seem reasonable to me, which I realize is quite a range.

Reply:
If the maximum changes while automation system is active, can that break things? You could have pDAO update it manually periodically. Otherwise yeah, makes sense to have them pick once at the start.

Reply:
I think it would be safe, even if automation is on.

Reply:
Imo the higher you go for "max", the less necessary/helpful the spec is (might as well not have an upper spec). The main concern is around "min" to ensure we have enough incentive for a sufficient amount of RPL staked. If we are using this spec that means there is a surplus share, and the general idea would be "pay enough to secure governance, and let the rest go to surplus share". The assumption is something along the lines of "we decided the mechanism used in surplus share provides the most value to RPL, so we want to give as much as possible to surplus share without sacrificing governance security (sufficient voter share)".

If 65% staked RPL is "enough" for governance, and the market staked 70%, then that means we are "over paying" voter share when it could be going to surplus share, so it should rebalance accordingly.

Also note that "staked RPL" doesn't equate to "level of turnout", you can stake RPL just for the yield but not be an active governance participant (not really what we are going for with "voter share"), so just because we end up with more staked RPL doesn't mean we are getting higher levels of voter turnout.

And finally this could all be useless if surplus revenue decision lands on "voter share" and we don't have a "surplus share" or automated targets here (and we may trend toward ~100% staked RPL)